### PR TITLE
test the error codes on the task routes + fix the missing error codes on the limit and from

### PR DESCRIPTION
**Pull request by irevoire**

### CODE REVIEW

**Review by loiclec — APPROVED** [review]

**Review by loiclec — APPROVED** [review]

**Comment by irevoire** [comment]
bors merge

**Review by dureuill — APPROVED** [review]
approving again after lo's approval.

bors merge

**Pull request merged**
